scripts/install/createDiagnosticTable.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace oat\taoClientDiagnostic\scripts\install;
+
+use oat\taoClientDiagnostic\model\storage\Sql;
+use Doctrine\DBAL\Schema\SchemaException;
+
+class createDiagnosticTable extends \common_ext_action_InstallAction
+{
+    public function __invoke($params)
+    {
+        $SqlService = new Sql();
```

Comment:
you should get sql storage from the service manager to ensure that the persistence you get is the right one